tests/screenshot/apps/components/Drawer.js

```diff
@@ -0,0 +1,10 @@
+import Drawer from '../../../../Drawer';
+import React from 'react';
+
+const DrawerTests = [
+	<Drawer>{[]}</Drawer>,
+	<Drawer noAnimation>{[]}</Drawer>,
+	<Drawer open>{[]}</Drawer>
```

Comment:
I might be better to add `noAnimation` to all Drawer components.
If not, we might easily get fails due to timing like Spinner. Spinner has animation, so it is hard to get successful screenshot test result.